src/gridfs-stream/download.ts

```diff
@@ -313,8 +315,8 @@ function init(stream: GridFSBucketReadStream): void {
         ? stream.s.filter._id.toString()
         : stream.s.filter.filename;
       const errmsg = 'FileNotFound: file ' + identifier + ' was not found';
-      const err = new Error(errmsg);
-      (err as any).code = 'ENOENT';
+      const err = new MongoDriverError(errmsg);
+      err.code = 'ENOENT';
```

Comment:
Just commenting so we remember to make a ticket for this.

Reply:
NODE-3338